src/ftw_cli/inference.py

```diff
@@ -243,40 +244,52 @@ def polygonize(input_fn, output_fn, simplify, overwrite):
     # read the input file as a mask
     with rasterio.open(input_fn) as src:
         input_height, input_width = src.shape
-        crs = src.crs.to_string()
-        profile = src.profile
-        transform = profile["transform"]
+        original_crs = src.crs.to_string()
+        transform = src.transform 
         mask = (src.read(1) == 1).astype(np.uint8)
-        #input_height, input_width = mask.shape
         polygonization_stride = 2048
         total_iterations = (input_height // polygonization_stride) * (input_width // polygonization_stride)
+        
+        # Define the equal-area projection using EPSG:6933
+        equal_area_crs = CRS.from_epsg(6933)
+
         with tqdm(total=total_iterations, desc="Processing mask windows") as pbar:
             for y in range(0, input_height, polygonization_stride):
                 for x in range(0, input_width, polygonization_stride):
                     new_transform = transform * Affine.translation(x, y)
                     mask_window = mask[y:y+polygonization_stride, x:x+polygonization_stride]
                     for geom, val in rasterio.features.shapes(mask_window, transform=new_transform):
                         if val == 1:
-
+                            geom = shapely.geometry.shape(geom)
                             if simplify is not None:
-                                geom = shapely.geometry.shape(geom)
                                 geom = geom.simplify(simplify)
+                            
```

Comment:
From memory, I think if you want to add the option to get rid of interior holes you can do:

```
if close_interiors:
    geom = shapely.geometry.Polygon(geom.exterior)
```

Reply:
Yep!

![image](https://github.com/user-attachments/assets/9efe5691-d885-4792-847c-5164ad16e1c1)

Reply:
Oh cool. Though I was hoping for something more nuanced - imagining that we'd let people set a threshold for removing holes. Maybe that's overkill - I was just thinking of it having work like the command I added to remove the tiny bits, where you set a threshold. 

I don't think there's actually fields that really have holes in them, so generally it's the right thing to do. But I'm imagining a case where there's some really big holes based on the raster, and user would prefer to see how it went wrong and get a closer representation of what the algorithm did.

But this is probably good enough for a first pass - we can make it an option so there's control over whether to do it or not.

Reply:
Added to #27